### ISSUE TITLE
module ATECC508A.Certificate.Template is not available

### ISSUE THREAD

**Issue by jfcloutier** [nerves-hub/atecc508a#43]
```
Upgraded:
  atecc508a 1.0.0 => 1.1.0

....

==> nerves_key
Compiling 7 files (.ex)
warning: ATECC508A.Certificate.Template.device/2 is undefined (module ATECC508A.Certificate.Template is not available or is yet to be defined)
Invalid call found at 2 locations:
  lib/nerves_key/data.ex:43: NervesKey.Data.slot_data/3
  lib/nerves_key/data.ex:98: NervesKey.Data.write_aux_certs/4

warning: ATECC508A.Certificate.Template.signer/1 is undefined (module ATECC508A.Certificate.Template is not available or is yet to be defined)
Invalid call found at 2 locations:
  lib/nerves_key/data.ex:38: NervesKey.Data.slot_data/3
  lib/nerves_key/data.ex:93: NervesKey.Data.write_aux_certs/4

warning: ATECC508A.Certificate.Template.device/2 is undefined (module ATECC508A.Certificate.Template is not available or is yet to be defined)
  lib/nerves_key.ex:116: NervesKey.device_cert/2

warning: ATECC508A.Certificate.Template.signer/1 is undefined (module ATECC508A.Certificate.Template is not available or is yet to be defined)
  lib/nerves_key.ex:141: NervesKey.signer_cert/2
```

**Comment by ConnorRigby** [comment]
I think this might require https://github.com/nerves-hub/nerves_key/pull/69 before that upgrade is allowed.

**Comment by fhunleth** [comment]
It turns out that 1.1.0 had a breaking change in it (the one reported). It should have been versioned 2.0.0. This issue is fixed in `nerves_key 1.1.0`, though, so if you update `nerves_key` now, you'll be fine.
